src/scripts/constants.ts

```diff
@@ -0,0 +1 @@
+export const EMBEDDED_COMPONENT_DOMAIN = 'https://dmc9najbd8k7f.cloudfront.net';
```

Comment:
If it's link to to hosted components I would change it to
HOSTED_COMPONENTS_DOMAIN

Reply:
good catch, thanks!